src/server/ApiRoutes/Search.js

```diff
@@ -1,10 +1,19 @@
-import { search } from '../../app/actions/SearchActions';
+import { searchGet, fetchWork } from '../../app/actions/SearchActions';
 import { getRequestParams } from '../../app/search/query';
 
 export const searchServer = (req, res, next) => {
-  const { q, filters } = getRequestParams(req.query);
+  const { q } = getRequestParams(req.query);
 
-  res.data = search(q, filters);
+  res.data = searchGet(q);
+
+  next();
+};
+
+export const getWorkById = (req, res, next) => {
+  const { workId } = req.query.workId;
```

Comment:
For this to work you want to destructure the object where the value is stored. So `const { workId } = req.query;` This is useful of `req.query` is an object with multiple properties and you only want to extract a few. If you only need `workId` you can do `const workId = req.query.workId` without the braces.

Reply:
Deconstructing properly would be better -- I'll adjust.